### PR TITLE
Promote separated sdn

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @sttts @deads2k 

/hold
to sync with https://github.com/openshift/origin/pull/23407

### CODE REVIEW

File: ci-operator/config/openshift/oc/openshift-oc-master.yaml [hunk 1]
```diff
@@ -19,10 +19,9 @@ images:
       as:
         - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
   to: cli
-# Do not promote until migrated from staging
-#promotion:
-#  name: "4.2"
-#  namespace: ocp
+promotion:
```
Comment (deads2k) [inline ci-operator/config/openshift/oc/openshift-oc-master.yaml]: wrong file?  SDN pull contains oc change?

Comment (tnozicka) [reply ci-operator/config/openshift/oc/openshift-oc-master.yaml]: thx, goland saved it from my previous PR

**Comment by tnozicka** [comment]
/hold cancel
we promote first to unblock https://github.com/openshift/cluster-network-operator/pull/248 and nuke after that and ART are done

**Pull request merged**